edvart/report_sections/multivariate_analysis.py

```diff
@@ -96,27 +99,18 @@ def __init__(
         )
 
         subsec = MultivariateAnalysis.MultivariateAnalysisSubsection
-        verbosities = {
+        self.subsection_verbosities = {
             subsec.PCA: verbosity_pca,
             subsec.ParallelCoordinates: verbosity_parallel_coordinates,
             subsec.ParallelCategories: verbosity_parallel_categories,
         }
         if UMAP_AVAILABLE:
-            verbosities[subsec.UMAP] = verbosity_umap
+            self.subsection_verbosities[subsec.UMAP] = verbosity_umap
 
-        # By default use all subsections
         if subsections is None:
-            subsections_all = list(subsec)
+            self.subsections_to_show = self._DEFAULT_SUBSECTIONS_TO_SHOW
         else:
-            subsections_all = subsections
-
-        # Store subsections with LOW verbosity
-        self.subsections_low_verbosity = [
-            sub for sub in subsections_all if verbosities[sub] == Verbosity.LOW
-        ]
-
-        if len(self.subsections_low_verbosity) == len(subsections_all) and subsections is None:
-            self.subsections_low_verbosity = None
+            self.subsections_to_show = subsections
```

Comment:
What about this? I can't decide myself which I like better :D
```python
self.subsections_to_show = (
    subsections
    if subsections is not None
    else self._DEFAULT_SUBSECTIONS_TO_SHOW
)
```

Reply:
Or `self.subsections_to_show = subsections or self._DEFAULT_SUBSECTIONS_TO_SHOW`?

Reply:
That doesn't work if subsections is an empty list `[]`. Not sure if this is a relevant use case, but better not to break it IMO.

Reply:
Good point.
I personally prefer if-else blocks instead of ternary operators. I'd keep it as it is.